plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -46,22 +48,22 @@ func ClusterRoles() []rbac.ClusterRole {
 	return []rbac.ClusterRole{
```

Comment:
Want to just create a filter method that adds all these `addBootstrappingLabel([]rbac.ClusterRole)` around the whole thing.